### ISSUE TITLE
Mysterious Song (2000)

### ISSUE THREAD

**Issue by matthew-goddard** [exoscoriae/eXoDOS#2503]
Metadata contains special characters which are not displaying properly in Launchbox. Suggestion to change "Notes" field to this for the supported formatting:

```
One of the better freeware "indie" RPGs ever made, Mysterious Song is a fun console-style RPGs that features hand-drawn 2D tile graphics, rendered battle backgrounds, nice original music score, and solid gameplay. Take on the role of Spear, and embark on a quest to discover your true identity.

True to its console influence, Mysterious Song is very easy to learn and play. You can equip your party members and check their stats via a console-style menu. The writing is generally good, although there is a few obvious typos and awkward sentences. Monsters are well-coded, and battles are generally well-balanced magic users and fighters are both essential to success. One of the game's high points is the plot development, which is full of surprising twists and turns throughout the very linear mission-based structure (i.e. do X for Y to get to Z).

In short, Mysterious Song is a competently coded amateur RPG that should keep console fans happy for a while. The game is a bit on the short side-10 hours or less of play time, but it's quite enjoyable throughout.
```

**Comment by exoscoriae** [comment]
done
